extensions/wikia/ImageReview/ImageReviewEventsHooks.class.php

```diff
@@ -80,4 +93,32 @@ private static function createAddTask( Title $title ) {
 		$task->prioritize();
 		$task->queue();
 	}
+
+	private static function sendToImageReveiwService( Title $title ) {
+		global $wgImageReview, $wgCityId;
+
+		$rabbitConnection = new \Wikia\IndexingPipeline\ConnectionBase( $wgImageReview );
```

Comment:
I don't think we should introduce tight coupling with IndexingPipeline extension. We should either create a generic rabbit connection wrapper class available globally and use it as needed, or create a local one for this extension.